README.md

```diff
@@ -94,30 +104,53 @@ $ grunt vendor
 $ grunt
 ```
 
+### 2. Run Elasticsearch
+
+To launch Elasticsearch, first find out where your Elasticsearch config file is located. You can do this with [Homebrew](http://brew.sh) using:
+
+```bash
+$ brew info elasticsearch
+```
+
+The last line of that output should be the command you need to launch Elasticsearch with the proper path to its configuration file. For example, it may look like:
+
+```
+$ elasticsearch --config=/Users/<YOUR MAC OSX USERNAME>/homebrew/opt/elasticsearch/config/elasticsearch.yml
+```
+
+### 3. Launch sheer to serve the site
+
 To work on the app you will need sheer running to compile the templates.
 There is also a `grunt watch` command that will recompile Less and JS
 on the fly while you're developing. To do this, run the following:
 
 ```bash
-# use the sheer virtualenv
+# Use the sheer virtualenv.
 $ workon sheer
 
-# index the latest content
+# Index the latest content from the API output of our WordPress back-end.
```

Comment:
If a new FEWD joins the team I would say that it would be required for them to be able to index from WP in order to work on the project. Pages that rely on querying content will error without this, but there are some dumb static HTML pages too. Overall, serious development of this repo requires having indexed content. I think it would be interesting though to have some dummy data that people could index to get up and running without having to hit WP.